package.json

```diff
@@ -11,33 +11,32 @@
 	},
 	"license": "MIT",
 	"devDependencies": {
+		"@aurelia/platform-browser": "^2.0.0-beta.19",
+		"@aurelia/testing": "^2.0.0-beta.19",
+		"@aurelia/ts-jest": "^2.0.0-beta.19",
+        "@aurelia/i18n": "^2.0.0-beta.19",
 		"@types/bootstrap": "^5.1.12",
 		"@types/jest": "^27.4.1",
 		"@types/sanitize-html": "^2.6.2",
-		"aurelia-binding": "^2.5.4",
-		"aurelia-bootstrapper": "^2.3.3",
-		"aurelia-dependency-injection": "^1.5.2",
-		"aurelia-event-aggregator": "^1.0.3",
-		"aurelia-loader-webpack": "^2.2.1",
-		"aurelia-polyfills": "^1.3.4",
-		"aurelia-webpack-plugin": "^4.0.0",
-		"jest": "^27.5.1",
-		"jest-html-reporters": "^3.1.4",
-		"ts-jest": "^27.1.4",
+		"apexcharts": "^3.49.1",
+		"aurelia": "^2.0.0-beta.19",
+		"echarts": "^5.5.0",
+		"jest-environment-jsdom": "^29.7.0",
```

Comment:
I think `jest` should added as dev dependency here, otherwise tests will not run:
`"jest": "^29.7.0",